### PR TITLE
make radify.sh ask for a base port

### PR DESCRIPTION
**Pull request by james-nesbitt**
This patch adds a "base port" question to the radify questions.

The answer will be used a a port prefix, to which will be appended single digit values for each service.

example:  if base port is 808, then:
   nginx : 8081
  varnish : 8080

This should deliver https://github.com/wunderkraut/radi-project-wundertoolswrapper/issues/17

### CODE REVIEW

**Comment by james-nesbitt** [comment]
@janne please review

File: .radi/init.yml [hunk 1]
```diff
@@ -235,6 +235,23 @@
     
     	echo " "
     fi
+    if [ -z "${PORTBASE}" ]; then
+    	echo "What is a good base port to use?
+    
+    	   This should be an integer like '808'
```
Comment (tharna) [inline .radi/init.yml]: I think this could be just xx80 and xx81, no need for the 3rd number?

Comment (james-nesbitt) [reply .radi/init.yml]: I will make that change

**Comment by james-nesbitt** [comment]
patch added to this PR for port interpretation.

@janne please review again

**Pull request merged**